src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -806,6 +813,21 @@ std::wstring_view Profile::GetName() const noexcept
     return _name;
 }
 
+// Method Description:
+// - Returns the tab title, if one is set. Otherwise returns the empty string.
+// Return Value:
+// - this profile's tab title, if one is set. Otherwise returns the empty string.
+std::wstring_view Profile::GetTabTitle() const noexcept
+{
+    return _tabTitle.has_value() ? std::wstring_view{ _tabTitle.value().c_str(), _tabTitle.value().size() } :

```

Comment:
What about when the user is using `name` as the tab title? Should just `name` +`suppressApplicationTitle:true` result in the profile's name suppressing the title?